src/main/java/dk/kb/storage/facade/DsStorageFacade.java

```diff
@@ -169,14 +169,41 @@ public static Long getRecordsByRecordTypeModifiedAfterWithLocalTree(
     }
     
     
+    
+    
+    /**
+     * Load a record with childrenIds and parentId if they exist 
+     *  If includeLocalTree is true also load the local tree for the given record. Parent will be loaded and all children. Siblings will not be loaded.  
+     *  1) If there is a parent record, the given record will point to it, but the parent will not point back to this child
+     *  2) Children will be loaded, but the children will not point back to this parent record.      
```

Comment:
To make this super pretty please use an HTML ordered list in the doc. This renders pretty in editors.
```
<ol>
<li></li>
</ol>
```